### PR TITLE
Add NR extension user fields to SCIM API

### PR DESCRIPTION
**Pull request by kdef-nr**
Note: @zuluecho9 is overseeing this. This PR is for updating the main SCIM API resource doc. Another PR https://github.com/newrelic/docs-website/pull/3239 is for a new tutorial and related edits. Jira: DOC-7815.

Add a table for the custom New Relic User schema extension.

The literal definition of the schema, as returned in  `/scim/v2/Schemas` endpoint
```
{
      "id": "urn:ietf:params:scim:schemas:extension:newrelic:2.0:User",
      "name": "New Relic User",
      "description": "New Relic User Schema",
      "attributes": [
        {
          "name": "nrUserType",
          "type": "string",
          "multiValued": false,
          "description": "The User's type, e.g., 'Full User'.",
          "required": false,
          "caseExact": false,
          "mutability": "readWrite",
          "returned": "default",
          "uniqueness": "none",
          "canonicalValues": [
            "Full User",
            "Basic User"
          ]
        }
      ],
      "meta": {
        "resourceType": "Schema",
        "location": "/Schemas/urn:ietf:params:scim:schemas:extension:newrelic:2.0:User"
      }
    }
```

This is an extension to the core User schema. This is visible in the updated User resource type definition as returned in the `/scim/v2/ResourceTypes` endpoint:
```
    {
      "schemas": [
        "urn:ietf:params:scim:schemas:core:2.0:ResourceType"
      ],
      "id": "User",
      "name": "User",
      "endpoint": "/Users",
      "description": "User",
      "schema": "urn:ietf:params:scim:schemas:core:2.0:User",
      "schemaExtensions": [
        {
          "schema": "urn:ietf:params:scim:schemas:extension:newrelic:2.0:User",
          "required": false
        }
      ],
      "meta": {
        "location": "/ResourceTypes/User",
        "resourceType": "ResourceType"
      }
    }
```

### CODE REVIEW

File: src/content/docs/accounts/accounts/automated-user-management/scim-support-automated-user-management.mdx [hunk 1]
```diff
@@ -18,19 +18,23 @@ Before using our SCIM API, you must first [enable SCIM for an authentication dom
 
 Note that after you enable SCIM, there are necessary [next steps](#next-steps) to do, including downgrading some users to basic users, and granting user groups access to New Relic accounts. 
 
+## Tutorial [#tutorial]
+
+If you want to see more specific instructions on how to use this API, see the [SCIM API tutorial](/docs/accounts/accounts/automated-user-management/tutorial-manage-users-groups-scim).
```
Comment (paperclypse) [inline src/content/docs/accounts/accounts/automated-user-management/scim-support-automated-user-management.mdx:23]: I'd rather front load this with the link. Something like:
"See the SCIM API tutorial for more specific information on using this API."

File: src/content/docs/accounts/accounts/automated-user-management/scim-support-automated-user-management.mdx [hunk 2]
```diff
@@ -170,6 +174,38 @@ New Relic uses a subset of the available fields in the SCIM core schema. Other S
   </tbody>
 </table>
 
+## New Relic user type (basic vs full) schema [#user-type]
+
+An optional [schema extension](https://datatracker.ietf.org/doc/html/rfc7643#section-3.3) for New Relic-specific user attributes. Currently this provides control only over a user's [user type](/docs/accounts/accounts-billing/new-relic-one-user-management/new-relic-one-user-model-understand-user-structure/#user-type) (basic user versus full user). For a tutorial on how to use this, see the [SCIM API tutorial](/docs/accounts/accounts/automated-user-management/tutorial-manage-users-groups-scim/#manage-user-type).
```
Comment (paperclypse) [inline src/content/docs/accounts/accounts/automated-user-management/scim-support-automated-user-management.mdx:179]: This first sentence is incomplete.

Comment (paperclypse) [reply src/content/docs/accounts/accounts/automated-user-management/scim-support-automated-user-management.mdx:179]: See above comment on front-loading the tutorial link.

File: src/content/docs/accounts/accounts/automated-user-management/scim-support-automated-user-management.mdx [hunk 3]
```diff
@@ -783,13 +819,16 @@ Items in this section could change as we work to bring our service provider into
 
 ## Next steps when you're done [#next-steps]
 
-### Downgrade users 
+### Downgrade users [#downgrade]
+
+When your users are provisioned in New Relic, you should then be able to see them in the [**User management** UI](/docs/accounts/accounts-billing/new-relic-one-user-management/add-manage-users-groups-roles/#where).
```
Comment (paperclypse) [inline src/content/docs/accounts/accounts/automated-user-management/scim-support-automated-user-management.mdx:824]: I'm leery of "should".... Are there times when this won't be true? What should someone do if they don't see users in the User management UI?

**Review by paperclypse — APPROVED** [review]
Looks pretty good. I left a couple comments.

**Review by zuluecho9 — DISMISSED** [review]

**Pull request merged**